### DISCUSSION TITLE
How to change format on click

### DISCUSSION THREAD

**Discussion by dhruvin-purohit** [Alexays/Waybar#1947] [Q&A] [answered]
I want to change format on click.
My goal is to cycle through date, day and month of clock module on-click.
I checked wiki but could not find how to change the format on-click.

**Comment by miranyx** [comment, marked as answer]
For example for date and time you could have the following. The first shows the date in ISO-8601 and on click the time:
```
"clock": {
    "format": "{:%F}",
    "format-alt": "{:%H:%M}"
}
